tests/test_cli/test_add_metadata.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+
+# -----------------------------------------------------------------------------
+# Copyright (c) 2011-2015, The BIOM Format Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from biom.cli import add_metadata
+from biom.parse import load_table
```

Comment:
`from biom import load_table`

...but, doesn't really matter for the time being